3.processing_features/scripts/2.annotate_sc.py

```diff
@@ -62,11 +62,43 @@
 pprint.pprint(plate_info_dictionary, indent=4)
 
 
+# ## Annotate bulk profiles
```

Comment:
I would consider changing the file names from `2.annotate_sc.py` to `2.annotate_bulk.py` to avoid confusion

Reply:
Thank you for bringing this up! I now see how it could be confusing. I have changed the names for annotate, normalize, and feature select to remove the `_sc` suffix.